server/src/main/java/org/apache/cloudstack/vm/UnmanagedVMsManagerImpl.java

```diff
@@ -692,8 +692,13 @@ private Pair<DiskProfile, StoragePool> importDisk(UnmanagedInstanceTO.Disk disk,
             chainInfo = gson.toJson(diskInfo);
         }
         StoragePool storagePool = getStoragePool(disk, zone, cluster);
+
+        if ( useControllerConfiguration ){
```

Comment:
```suggestion
        if (useControllerConfiguration) {
```

Reply:
@DK101010 , this is one of the code smells and it is easily fixed:
```suggestion
        if (Boolean.TRUE.equals(useControllerConfiguration)) {
```